### ISSUE TITLE
onChange event not fired when date picked from datepicker

### ISSUE THREAD

**Issue by indreksiitan** [fomantic/Fomantic-UI#1251]
Expected behaviour:
* When a date is picked from the datepicker in the Calendar component, the "change" event is fired on the updated input.

Actual behaviour:
* When I manually modify the date and navigate away from the field (blur the input), the change event is fired as expected.
* When I choose a date from the Calendar datepicker, change event is not fired neither immediately nor navigating away from the field.
* When I manually modify the date in the input, then open the datepicker, close it without picking anything and then blur the field, change event does not occur.

Tested with 2.8.3.

**Comment by evandromendonca** [comment]
@indreksiitan I am using 2.4.1 but having this same problem. What I did to go around this was calling dispatchEvent inside the onChange event that the calendar component defines.
```
$('#element_id')
    .calendar({
        ...,
        onChange: function (date, text) {                        
            document.querySelector('#element_id input').dispatchEvent(new Event('change'));
        }
    });
```

It doesn't seem ideal but this worked for me as a quick fix. I hope it helps.

**Comment by lubber-de** [comment]
Fixed by #1602
